### DISCUSSION TITLE
Question: How calculate maxZoom?

### DISCUSSION THREAD

**Discussion by undefineq** [ValentinH/react-easy-crop#292] [Q&A]
Hello,

Don't want users to be able to zoom in less than 1000 pixels. What I mean by that is when users zoom in the image size, suppose 1200 pixels, it can't be lower than 1000 pixels. I assume that maxZoom is for such a case, but I can't figure out how to use it? Can you please clarify me how it was originally intended?

**Comment by ValentinH** [comment]
The `maxZoom` property takes the same kind of value that the one you pass to `zoom`. This is a zoom factor. 2 means the image size will be twice as small as the original one.
